py/occampy3/wrappers/model.py

```diff
@@ -55,3 +63,12 @@ def set_id(self, id_: int) -> None:
 
     def set_progenitor(self, progenitor: 'Model') -> None:
         self._ref.setProgenitor(progenitor.ref)
+
+    def get_attribute_value(self, attribute: str) -> Union[str, float]:
+        return self._ref.get(attribute)
+
+    def is_is_equivalent_to(self, model: 'Model') -> bool:
```

Comment:
Could define `__eq__` in this way as well, to enable comparing models using ==.

Reply:
Not sure if in this case equivalence would be the same as equality

Reply:
I think it works here, as `==` implies functional equivalence while `is` implies being literally the same.

Reply:
In my understanding `==` would mean that both `model` objects are the same, meaning that they have the same name, values, etc.. But yes, I looked in the C++ code and it does actually compare them with the `==` operator